birdhouse/scripts/logging.include.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
-BIRDHOUSE_COLOR=${BIRDHOUSE_COLOR:-1}
+# support NO_COLOR flag (https://no-color.org/)
+if [ -z "${BIRDHOUSE_COLOR}" ] && [ "${NO_COLOR}" != '1' ]; then
```

Comment:
```suggestion
if [ -z "${BIRDHOUSE_COLOR}" ] && [ -z "${NO_COLOR}" ]; then
```

Reply:
https://no-color.og/ says "NO_COLOR environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color."  so I think we should not just check for `NO_COLOR == 1`.